### PR TITLE
Fix usability problems

### LINKED ISSUE #14: Turn off "feedback mask"
Currently, if one writes a program that compiles but has warnings, say,
```haskell
import Prelude hiding (rotated,translated,colored,lettering,scaled,polyline,Text,Number)

program = drawingOf(picture & coordinatePlane)

picture = circle(1)

a = 1
```
one gets this kind of link at the warning messages: 
image

Clicking on "Not helpful?" opens this window: 
image

I'm not sure what/where would happen if I entered something there and clicked "Ok". But I guess it's not something we will actually get/read, so students might be confused if they report something there but there is never any reaction.

### LINKED ISSUE #15: Display of `path` in code window
Let's take this simple program:
```haskell
import Prelude hiding (rotated,translated,colored,lettering,scaled,polyline,Text,Number)

program = drawingOf(picture & coordinatePlane)

picture = path([(0,0),(5,5)])
```
For some reason, the last line looks as follows on the website: 
image

For comparison, on the original site is looks as follows: 
image

### PR DESCRIPTION
**Pull request by nimec01**
Closes #14 
Closes #15

### CODE REVIEW

File: web/js/codeworld_shared.js [hunk 1]
```diff
@@ -1485,79 +1485,79 @@ function printMessage(type, message) {
...
+
+  //     if ($lastOutputBlockMessageContent.children().length) {
+  //       $lastOutputBlockMessageContent
+  //         .find('.message-summary')
+  //         .append($reportLink);
+  //     } else {
+  //       $lastOutputBlockMessageContent.append($reportLink);
+  //     }
+  //   }
+  // }
```
Comment (marcellussiegburg) [inline web/js/codeworld_shared.js]: Would a block comment be better here to reduce diff noise?

File: web/js/codeworld_shared.js [hunk 2]
```diff
@@ -1485,79 +1485,79 @@ function printMessage(type, message) {
...
+//     sweetAlert({
+//       type: 'success',
+//       text: 'Thank you for your feedback.',
+//     });
+
+//     $reportLink.hide();
+//     window.alreadyReportedErrors.add(scrubError(message));
+//   });
+//   event.preventDefault();
+// }
```
Comment (marcellussiegburg) [inline web/js/codeworld_shared.js]: (same as above)

File: web/js/codeworld_shared.js [hunk 3]
```diff
@@ -1485,79 +1485,79 @@ function printMessage(type, message) {
...
-function scrubError(msg) {
-  return msg
-    .replace(/[a-zA-Z0-9_-]+\.hs:(\d+):((\d+)(-\d+)?)/g, '(loc)')
-    .replace(/[a-zA-Z0-9_-]+\.hs:\((\d+),(\d+)\)-\((\d+),(\d+)\)/g, '(loc)');
-}
+// function scrubError(msg) {
+//   return msg
+//     .replace(/[a-zA-Z0-9_-]+\.hs:(\d+):((\d+)(-\d+)?)/g, '(loc)')
+//     .replace(/[a-zA-Z0-9_-]+\.hs:\((\d+),(\d+)\)-\((\d+),(\d+)\)/g, '(loc)');
+// }
```
Comment (marcellussiegburg) [inline web/js/codeworld_shared.js]: (same as above)

File: web/js/codeworld_shared.js (hunk 1 above)
Comment (marcellussiegburg) [reply web/js/codeworld_shared.js]: @nimec01 Or is there a specific reason for line comments?

Comment (nimec01) [reply web/js/codeworld_shared.js]: I changed it

**Review by marcellussiegburg — APPROVED** [review]

**Pull request merged**